technic/machines/register/cables.lua

```diff
@@ -178,11 +178,12 @@ local function override_table(target, source)
 	return target
 end
 
-function technic.register_cable(tier, size, description, prefix, override_cable, override_cable_plate)
+function technic.register_cable(tier, size, description, prefix, override_cable, override_cable_plate, modname)
+	modname = modname or minetest.get_current_modname()
```

Comment:
This argument list does not look very nice and is getting worse, there must be some better way to bring custom modname here.
For custom mod name maybe think again and add variable to technic namespace for mod name, would that work?
What is actual use case?

Reply:
IMO this should be changed to be like the other register functions, to use a `data` table. It doesn't make sense to have one register function work differently to the rest...

Reply:
> should be changed to be like the other register functions, to use a data table

Very much agreed :+1: but not in this PR. Should still avoid adding to list...